cosmos/templates/gmm/gmm_update.html

```diff
@@ -5,8 +5,11 @@
 {% block content %}
 
 <div class="container">
-    <div class="mx-auto w-50">
-        <form method="post" enctype="multipart/form-data" id="form-container">
+    <div class="mx-auto w-50 bg-cosmos-primary rounded">
+        <form method="post" enctype="multipart/form-data" id="form-container" class="p-4 overflow-hidden">
+
+            <h2 class="text-white">Update gmm documents</h2>
+
             {% csrf_token %}
             {% crispy form %}
```

Comment:
The black form labels are now illegible with a dark background

![image](https://user-images.githubusercontent.com/19555626/127763584-afdfd3a7-3a8c-4bda-a6a1-a9dfc2065122.png)